src/bioregistry/data/bioregistry.json

```diff
@@ -4140,7 +4140,7 @@
       "synonyms": [
         "BIORXIV"
```

Comment:
Curious why there is an ALL CAPS synonym? Shouldn't synonyms be distinct in a case-insensitive way... i.e. a `BIORXIV` prefix is already supported without this.

Reply:
I've curated the synonyms in a case-sensitive way - so this reflects what I've seen in the wild. Internally, the bioregistry throws away case sensitivity when it does lookup.